daemon/containerd/image_push.go

```diff
@@ -19,6 +20,7 @@ import (
 	"github.com/distribution/reference"
 	"github.com/docker/docker/api/types/events"
 	"github.com/docker/docker/api/types/registry"
+	dimages "github.com/docker/docker/daemon/images"
```

Comment:
A bit related to that, and I didn't really consider that's already the case, but the `containerd` and `images` packages are the two implementations we have for handling the image-store. I see there's multiple imports of the `images` package into the `containerd` implementation. Not "critical", but it feels a bit odd. Perhaps we should look at what parts are used "in common" and if that code is in the right place (or should be in a "common" package that's shared between the two if duplicating those parts is undesirable)